core/server/services/url/utils.js

```diff
@@ -238,7 +249,7 @@ function urlFor(context, data, absolute) {
         // this will become really big
         knownPaths = {
             home: '/',
-            api: API_PATH,
+            api: getApiPath('stable'),
```

Comment:
Is knownPath with `api` used? 🤔

Reply:
Good point, there is a general `knownPaths[context]`, but it seems anyways when `context = 'api'` code will never reach that point. Not sure why its there in first place, I'll remove it from knownPaths if there was no particular historic reason as it seems its not useful.